canonical/ubuntu.com
# WD-9474 Refresh download core

## Done

- Refresh Ubuntu Core download page
- Redirect old download/iot pages to -> download/core
- Update /download/ page with the correspoding IoT -> Core replacement
- Refactor blog partial

## QA

- Go to https://ubuntu-com-13740.demos.haus/download/core and compare with [copy docs](https://docs.google.com/document/d/1Sxdl5cGGe1wbHmKB2eL2zQLfVW3KBea02a2FIGx3VdQ/edit#heading=h.6lhihgdcnq6v) and compare with design (screenshot below) [Before Core 22 only]
    - Be sure to test on mobile, tablet and desktop screen sizes

- Old page https://ubuntu-com-13740.demos.haus/download/iot should redirect to -> https://ubuntu-com-13740.demos.haus/download/core
- Test https://ubuntu-com-13740.demos.haus/download/desktop page works after blog section refactorization. Click on Download 24.04 to check that the thank-you page stills works, the download won't work because we still haven't released Ubuntu 24.04, but the rest of the page should work.

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-9479
Fixes https://warthogs.atlassian.net/browse/WD-10106

## Screenshots (design review)

![core](https://github.com/canonical/ubuntu.com/assets/14939793/e643a6cf-9d83-4bfe-b6f6-bbda94ed2bdc)


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
